chain/store/account.go

```diff
@@ -224,13 +254,23 @@ func (sdb *StateDB) GetNonce(addr common.Uint160) uint64 {
 	return account.GetNonce()
 }
 
-func (sdb *StateDB) GetID(addr common.Uint160) []byte {
+func (sdb *StateDB) GetID(addr common.Uint160, height uint32) []byte {
```

Comment:
We need to add GetID1 and GetID2 as well. Then just choose to call one of them in GetID based on height